src/main/java/freiman/citibike/aws/CitibikeRequestHandler.java

```diff
@@ -1,27 +1,51 @@
-//package freiman.citibike.aws;
-//
-//import com.amazonaws.services.lambda.runtime.Context;
-//import com.amazonaws.services.lambda.runtime.RequestHandler;
-//import software.amazon.awssdk.services.s3.S3Client;
-//import software.amazon.awssdk.services.s3.model.GetObjectRequest;
-//
-//import java.io.IOException;
-//import java.io.InputStream;
-//
-//public class CitibikeRequestHandler {
-//    private final S3Client s3Client;
-//
-//    public CitibikeRequestHandler() {
-//        // Initialize the s3 client
-//        s3Client = S3Client.create();
-//    }
-//
-//}
-//
-//
-//
-//
-//
-//
-//
-//
+package freiman.citibike.aws;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import freiman.citibike.json.StationService;
+import freiman.citibike.json.StationServiceFactory;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.GetObjectRequest;
+import freiman.citibike.StationFinder;
+import freiman.citibike.json.Station;
+import java.util.Map;
+
+public class CitibikeRequestHandler implements
+        RequestHandler<CitibikeRequest, CitibikeResponse> {
```

Comment:
Unfortunately this doesn't work the way I thought it did.
Refer to my updated documentation
https://github.com/andrewoid/evolution-java/blob/main/amazon/aws_lambda.md